vtr_flow/misc/synthesis.ys

```diff
@@ -0,0 +1,81 @@
+# XXX is replaced with filename by the run_vtr_flow script
```

Comment:
Should comment what this script is for: flow script for yosys synthesis (put that before the line saying "XXX is replaced by"